pkg/cmd/admin/registry/registry.go

```diff
@@ -74,7 +75,14 @@ type RegistryConfig struct {
 
 var errExit = fmt.Errorf("exit")
 
-const defaultLabel = "docker-registry=default"
+const (
+	defaultLabel = "docker-registry=default"
+	defaultPort  = 5000
+	// TODO: `/healthz` has been deprecated by `/`; keep it temporarily for
+	// backwards compatibility
+	healthzRoute               = "/healthz"
```

Comment:
> Shall I make the comment more specific about the when?

I was just worried about mixing a new URL with an old image.  Does the v1 image support "/"?  If not, you'll need to be sure you don't add an invalid probe to an older image.